lib/array_validator/order_validator.rb

```diff
@@ -0,0 +1,19 @@
+class ArrayValidator
+  module OrderValidator
+    def self.call(record, attribute, values, options)
+      return if options[:order].nil?
+      raise ArgumentError, 'Not a supported order option' unless [:asc, :desc].include?(options[:order])
```

Comment:
i'd just change this to `options[:order].in?([:asc, :desc])` and that's it.

I'm surprised that Rails does not have `sort(:asc)` and `sort(:desc)`.